tests/unit/AGMDate.spec.ts

```diff
@@ -24,8 +24,8 @@ describe('AgmDate', () => {
     store.state.ARFilingYear = 2019
     store.state.arMinDate = '2019-01-01'
     store.state.arMaxDate = '2019-12-31'
-    store.state.business.legalType = 'CP'
-    store.state.business.lastAnnualReportDate = '2018-07-15'
+    store.commit('setLegalType', 'CP')
+    store.commit('setLastAnnualReportDate', '2018-07-15')
```

Comment:
Not sure if you want to review the remaining unit tests but it's this change in all of them -- all changes to store.state.business now uses commits.